server/routerlicious/packages/routerlicious-base/src/riddler/api.ts

```diff
@@ -113,6 +119,18 @@ export function create(
 		handleResponse(storageP, response);
 	});
 
+	/**
+	 * Updates the keyless access setting for the given tenant
+	 */
+	router.put("/tenants/:id/keylessAccess", (request, response) => {
+		const tenantId = request.params.id;
+		const enableKeylessAccess = request.body.enableKeylessAccess
+			? request.body.enableKeylessAccess
+			: null;
+		const storageP = manager.updateKeylessAccessPolicy(tenantId, enableKeylessAccess);
```

Comment:
We should be able to update both enableKeylessAccess and enableKeyAccess. Ensure that both cannot be set to false.

Reply:
Ahh okay got it. So in case both are true, then we will have `(key, secondaryKey)` present for the tenant. In case only `enableKeyAccess` is true, then we would only have `(key, secondaryKey)` present. In case only `enableKeylessAccess` is true, then we would only have `privateKeys` present for the tenant. Is my understanding correct?